timeless/__init__.py

```diff
@@ -19,11 +19,14 @@
 from timeless.restaurants.floors import views as floors_views
 from timeless.restaurants.table_shapes import views as table_shapes_views
 
+from flask_cache import Cache
+
 
 def create_app(config):
     """Creates a new Timeless webapp given a config class"""
     app = Flask(__name__, instance_relative_config=True)
     app.config.from_object(config)
+    cache = Cache(app, config={'CACHE_TYPE': 'redis'})
```

Comment:
@ammaratef45 At least pls update README how to set up redis.
But actually we have a Dockerfile and I prefer to leave here puzzle to set up redis to `docker-compose.yaml`.